Cargo.toml

```diff
@@ -59,11 +60,12 @@ experimental = ["static-alloc"]
 either = "1.5"
 inkwell_internals = { path = "./internal_macros", version = "0.1.0" }
 libc = "0.2"
-llvm-sys = "80.1"
+llvm-sys = "90.0.0"
 once_cell = "1.2"
 parking_lot = "0.9"
 regex = "1"
 static-alloc = { version = "0.1", optional = true }
+cfg-if = "0.1"
```

Comment:
We've been getting a lot of complaints about compilations times. I don't think we should add `cfg-if` since the stuff you use it for look like they should be able to be done easily without it.